### PR TITLE
[COST-20] Openshift virt

### PR DESCRIPTION
**Pull request by myersCody**
## Jira Ticket

[COST-20](https://issues.redhat.com/browse/COST-20)

## Description

This change will add a new UI monthly view for OpenShift virtualization. 

## Release Notes
- [ ] proposed release note

```markdown
* [COST-20](https://issues.redhat.com/browse/COST-20) Fix some things
```

### CODE REVIEW

**Comment by myersCody** [comment]
Smoke tests running here: https://ci.ext.devshift.net/job/koku-pipeline-pr-check-main/3933/console

File: koku/api/query_params.py [hunk 1]
```diff
@@ -452,7 +452,17 @@ def _set_time_scope_defaults(self):
...
+            self.set_filter(
+                time_scope_value=(
+                    time_scope_value if time_scope_value in TIME_SCOPE_VALUES_MONTHLY else TIME_SCOPE_VALUES_MONTHLY[0]
+                ),
+                time_scope_units=TIME_SCOPE_UNITS_MONTHLY,
+                resolution=RESOLUTION_MONTHLY,
+            )
+
         if self.report_type == "ec2_compute":
+            # TODO: Fix ec2 view to have a default_scope as well
```
Comment (lcouzens) [inline koku/api/query_params.py:465]: Do we still need to do this or is it a follow up thing?

Comment (myersCody) [reply koku/api/query_params.py:465]: I will do it in a follow up.

Comment (myersCody) [reply koku/api/query_params.py:465]: https://github.com/project-koku/koku/pull/5366

**Pull request merged**